meeting_minutes/2024/2024-05-29.md

```diff
@@ -22,18 +22,17 @@ Quorum was not reached due to inability to register attendees due to OASIS syste
 | Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member               |
 | Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member               |
 | Rhonda     | Levy        | Cisco                                                       | Voting Member               |
-| Christoph   | Plutte      | Ericsson                                                    | Member                      |
+| Christoph  | Plutte      | Ericsson                                                    | Member                      |
```

Comment:
Wasn't Tobias Limmer also there? At least he is mentioned in the protocol later on...

Reply:
@tlimmer Please review